### PR TITLE
NAXE new locus

### PR DESCRIPTION
**Pull request by laurelhiatt**
woohoo!

### CODE REVIEW

File: data/STRchive-database.json [hunk 1]
```diff
@@ -3855,10 +3705,98 @@
...
+       ],
+       "prevalence":null,
+       "prevalence_details":"Proband found in Japanese cohort [@doi:10.1038/s41525-024-00429-5].",
+       "stripy":[],
+       "gnomad":[],
+       "genereviews":[],
+       "mondo":[
+           "0020781"
+       ],
+       "year":"2024 [@doi:10.1038/s41525-024-00429-5]",
```
Comment (hdashnow) [inline data/STRchive-database.json:3775]: Did you discuss adding citations to year with @vincerubinetti? Just want to make sure we're not going to mess up parsing down the track. This would have flustered my old gene page scripts, but might be fine with the new website build.

File: data/STRchive-database.json [hunk 2]
```diff
@@ -3855,10 +3705,98 @@
...
+       "age_onset":"Predominately infantile onset, extending to 20y or older for general disease; proband onset at one year [@doi:10.1038/s41525-024-00429-5].",
+       "age_onset_min":1,
+       "age_onset_max":1,
+       "typ_age_onset_min":0,
+       "typ_age_onset_max":0,
+       "novel":"ref",
+       "mechanism":"LoF",
+       "mechanism_detail":"Reduced NAXE expression from expansion in promoter [@doi:10.1038/s41525-024-00429-5]",
+       "source":[],
+       "details":"Normal range (2-7) alleles established by 484 control alleles and validated with orthogonal databases, while proband had expansion of ~200 repeats [@doi:10.1038/s41525-024-00429-5].",
```
Comment (hdashnow) [inline data/STRchive-database.json:3763]: Suggest adding something like: locus is supported by a single individual with two expanded alleles inherited from their mother via uniparental disomy. While the repeat expansion is newly reported, other variants in the NAXE gene have previously been associated with mitochondrial encephalopathy.

File: data/STRchive-database.json [hunk 3]
```diff
@@ -3855,10 +3705,98 @@
...
+       "omim":[
+           "617186"
+       ],
+       "prevalence":null,
+       "prevalence_details":"Proband found in Japanese cohort [@doi:10.1038/s41525-024-00429-5].",
```
Comment (hdashnow) [inline data/STRchive-database.json:3768]: Single proband

File: data/STRchive-database.json (hunk 2 above)
Comment (hdashnow) [reply data/STRchive-database.json:3763]: Also normal -> benign

File: data/STRchive-database.json [hunk 4]
```diff
@@ -3855,10 +3705,98 @@
...
+       "location_in_gene":null,
+       "benign_min":2.0,
+       "benign_max":7.0,
+       "intermediate_min":null,
+       "intermediate_max":null,
+       "pathogenic_min":200.0,
+       "pathogenic_max":200.0,
+       "ref_copies":null,
+       "motif_len":5,
+       "age_onset":"Predominately infantile onset, extending to 20y or older for general disease; proband onset at one year [@doi:10.1038/s41525-024-00429-5].",
```
Comment (hdashnow) [inline data/STRchive-database.json:3754]: I'd reorder to emphasis what is known about the TR disease specifically first. E.g. something like: 
Single repeat expansion case had age of onset at 1 year, 1 month. For NAXE-related mitochondrial encephalopathy more generally, predominately infantile onset, extending to 20y or older.

File: data/STRchive-database.json (hunk 2 above)
Comment (hdashnow) [reply data/STRchive-database.json:3763]: Suggest adding something about this here and/or under mechanism:
"bioinformatics analysis of the kinetics features of PacBio long-read sequencing data detected CpG hypermethylation at the repeat sequence (GGGCC) per se and in the region downstream of the repeat sequence (Fig. [4c, d](https://www.nature.com/articles/s41525-024-00429-5#Fig4)) around the promoter region of the patient’s genome, compared to those in the controls. The long-read sequencing in the mother also showed hypermethylation in the allele with repeat expansion, whereas the allele with a normal range of GGGCC repeat did not show hypermethylation in the repeat and its downstream region (Fig. [4d](https://www.nature.com/articles/s41525-024-00429-5#Fig4))."

**Review by hdashnow — CHANGES_REQUESTED** [review]
Looks good. I left some comments on specific bits.

I'm overall not 100% sold on this locus based on this paper alone. I think it's a good candidate locus, but they haven't completely convinced me because:
- A single case
- The UPD makes it tricker to disentagle the TR from the surrounding sequence.
- All their functional work was done with patient cells, so it's impossible to know if there is a different variant that is actually causing it. It would be more convincing if they'd put the repeat into a different background.

In summary, the mechanism makes sense, it's probably real, but there's a chance there's another causal variant that the TR is just associated with.

**Comment by hdashnow** [comment]
Also, remind me to help you with rebasing before merging this.

File: data/STRchive-database.json (hunk 1 above)
Comment (laurelhiatt) [reply data/STRchive-database.json:3775]: I'm giving him a list of the fields that can have free text citations. we should be okay
